cocotb/share/lib/gpi_log/gpi_logging.c

```diff
@@ -99,6 +99,41 @@ const char *log_level(long level)
 #define LOG_SIZE    512
 static char log_buff[LOG_SIZE];
 
+
+/** Log as a backup, if python isn't working */
+static void gpi_log_native_v(const char *name, long level, const char *pathname, const char *funcname, long lineno, const char *msg, va_list argp) {
+    if (level >= GPIInfo) {
```

Comment:
Please invert the condition and return early to reduce the indentation of this block.

```c
if (level < GPIInfo) {
    return;
}
```

Reply:
This was done this way to match what was here before - I'll add an extra commit with this change.